rules-topics/04_combat.md

```diff
@@ -112,7 +116,7 @@ The target of a killing blow may refuse it, so long as they are capable of movem
 ---
 Rarely, you may encounter entities that are capable of delivering magic quicker than normal, skipping the incantation and abridging it to simply Magic followed by the spell name, for example, Magic Fear. This is still an effective spell, and needs to be reacted to as such.
 
-Similarly, some entities may deliver a spell via "Voice Radius," at which time anyone who can hear the declared spell, whether enemy or ally, is subject to it as if they’d been struck by the spell directly.
+Similarly, some entities may deliver a spell via "Voice Radius," at which time anyone who can hear the declared spell, whether enemy or ally, is subject to it as if they’d been struck by the spell directly. This type of effect does not require the caster to have a Spell Packet in hand.
```

Comment:
> This type of effect does not require

VR is a delivery method, not an effect. Should be something like:

Voice Radius does not require the caster to have a spell packet, and may be used without a free hand.